### PR TITLE
validate github username

### PR DESCRIPTION
**Pull request by abhishek97**
In reference to issue #9 

Note : uses node module `github` instead of just pinging `github.com/<username>` . The module will help in validating further and performing other checks ( check if pr is merged or if it exists etc.. )

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,11 @@
 {
   "dependencies": {
+    "bluebird": "^3.5.0",
```
Comment (championswimmer) [inline package.json]: can we not have 20 promise libraries in this project ? ES6 has promises, then we have RSVP already, and you are adding Bluebird. Quintessential nodejs dependency bucket ?

**Comment by abhishek97** [comment]
sorry for that. Sticking to ES6/RSVP only from now on.

File: utils/datautils.js [hunk 2]
```diff
@@ -49,20 +60,19 @@ function updateClaim(claimId, status) {
...
-        user,
-        issueUrl,
-        pullUrl,
-        repo: pullUrl.split('github.com/')[1].split('/')[1],
-        bounty: bounty,
-        status: status
-    })
+    return github.users.getForUser({
+        username : user
+    }).then(data=>{
```
Comment (championswimmer) [inline utils/datautils.js]: have you tested this as working ? 

aren't you returning the wrong promise form here

Comment (championswimmer) [inline utils/datautils.js]: @abhishek97 did you check this ?

**Comment by abhishek97** [comment]
@championswimmer yes i have checked this.

File: utils/datautils.js [hunk 3]
```diff
@@ -49,20 +60,19 @@ function updateClaim(claimId, status) {
 
 function createClaim(user, issueUrl, pullUrl, bounty, status) {
 
-    const claim = {
-        action: 'create',
-        user, issueUrl, pullUrl, bounty, status
-    };
-    fs.writeFile(__dirname + '/../audit/' + new Date().toISOString() + '.json', JSON.stringify(claim), () => {});
```
Comment (championswimmer) [inline utils/datautils.js]: don't remove the auditing  please

Comment (abhishek97) [inline utils/datautils.js]: @championswimmer  sorry, it must have been removed during a conflict. Restored now

**Comment by omerjerk** [comment]
Closed because we have a "better" alternative.

**Pull request closed without merging**